docs/workers.md

```diff
@@ -251,6 +257,23 @@ expressions:
     ^/_matrix/client/(api/v1|r0|v3|unstable)/join/
     ^/_matrix/client/(api/v1|r0|v3|unstable)/profile/
 
+    # Typing requests
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/typing
```

Comment:
This isn't true unless the worker is configured to handle typing? The idea with this list of regexes is that you can just straight up copy and paste this into your load balancers

Reply:
Is this kind of the same conversation as https://github.com/matrix-org/synapse/pull/11599#discussion_r779768039 then?

Reply:
I see, this endpoint does get registered but doesn't work unless it is the typing stream.